protocols/raft/src/main/java/io/atomix/protocols/raft/roles/CandidateRole.java

```diff
@@ -124,6 +124,7 @@ private void sendVoteRequests() {
         raft.transition(RaftServer.Role.FOLLOWER);
       }
     });
+    this.quorum = quorum;
```

Comment:
quorum was never set so if electrion times out the previous quorum was never cancelled.

Reply:
@kuujo not sure if this is quite right, wdyt?